package.json

```
@@ -19,7 +19,7 @@
     "lint": "eslint -c .eslintrc.json ."
   },
   "peerDependencies": {
-    "acorn": "^6.1.0"
+    "acorn": ">=6.1.0"
```

```suggestion
    "acorn": ">=6.1.0 <8.0.0"
```

@adrianheine What do you think of this?

---

That would work :)